### ISSUE TITLE
Roads are actually grass floor tiles

### ISSUE THREAD

**Issue by QuakeIV** [Lykayos/Fallout-13-EN#32]
Currently you can crowbar the roads and get grass floor tiles.  Also, if you bomb them hard enough they turn into plating.  This is not ideal.

**Comment by desertofunknown** [comment]
Yeah we know when the russian devs made the sprites they just copied grasstiles in the map maker and just changed the name and sprites

**Comment by QuakeIV** [comment]
Yeah, this is just so that someone hopefully actually fixes it.
